guidance/_utils.py

```diff
@@ -70,19 +90,33 @@ def __exit__(self, exc_type, exc_value, traceback):
         if self.restore_stderr is not None:
             sys.stderr._original_stdstream_copy = self.restore_stderr
 
+
 def strip_multiline_string_indents(f):
 
     source = textwrap.dedent(inspect.getsource(f))
-    blanks = '\n' * f.__code__.co_firstlineno # padd the source so the lines in the file line up for the debugger
-    source = blanks + '\n'.join(source.splitlines()[1:]) # remove the decorator first line.
-    
+    blanks = (
+        "\n" * f.__code__.co_firstlineno
+    )  # padd the source so the lines in the file line up for the debugger
+    source = blanks + "\n".join(
+        source.splitlines()[1:]
+    )  # remove the decorator first line.
```

Comment:
in general, I'm not a fan of how `black` handles long-ish lines with trailing comments. I suppose I'd prefer that it moves trailing comment lines to the preceding line, and then keep the source as a single compact line. But a standard is a standard 🤷‍♂️ . Would be nice for the future if there's a way to configure this in comments, as I think:
```python
 # padd the source so the lines in the file line up for the debugger
 blanks = "\n" * f.__code__.co_firstlineno
```
looks much nicer than
```python
 blanks = (
        "\n" * f.__code__.co_firstlineno
    )  # padd the source so the lines in the file line up for the debugger
```

Reply:
@riedgar-ms @ryanpeach curious about your thoughts here

Reply:
I would agree with your preference, although it's more something that could be fixed up as people review the code afterwards rather than needing to go through all the files now.